src/Approximations/overapproximate.jl

```diff
@@ -762,9 +762,16 @@ using .TaylorModels: Taylor1, TaylorN, TaylorModelN, TaylorModel1,
                      normalize_taylor, linear_polynomial,
                      constant_term, evaluate, mid
 
-# helper functions
-@inline get_linear_coeffs(p::Taylor1) = linear_polynomial(p).coeffs[2:end]
-@inline get_linear_coeffs(p::TaylorN) = linear_polynomial(p).coeffs[2].coeffs
+@inline function get_linear_coeffs(p::Taylor1, n)
+    return linear_polynomial(p).coeffs[2:end]
```

Comment:
Just to make sure: Here you do not need to check `p.order == 0`?

Reply:
Ok, i've added that case as well.